sdk/lib/crt/except/except.cmake

```diff
@@ -4,9 +4,13 @@ if(ARCH STREQUAL "i386")
         except/i386/chkstk_asm.s
     )
     list(APPEND CRT_EXCEPT_ASM_SOURCE
+        except/i386/__CxxFrameHandler3.s
         except/i386/chkesp.s
         except/i386/prolog.s
     )
+    list(APPEND CRT_EXCEPT_SOURCE
+        except/i386/CxxHandleV8Frame.c
+    )
```

Comment:
```suggestion
        except/i386/CxxHandleV8Frame.c)
```

Reply:
no

Reply:
Note:
I don't really mind either way, I am simply wondering why some parts have the parenthesis on the same line and some on the next line.